tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.jobs/Functions.cs

```diff
@@ -115,8 +116,10 @@ public class Functions
                             SecureString password = GetEncryptedPassword(configuration["Settings:AdminPassword"]);
                             ctx.Credentials = new SharePointOnlineCredentials(configuration["Settings:AdminUserName"], password);
                             //First check whether the user exists in SharePoint or not
+                            log.WriteLine($"Checking whether the user {email} has been present in the system or not");
```

Comment:
Added logging statements so that we can see this log information in the azure web job dashboard